### PR TITLE
Issue16

### PR DESCRIPTION
**Pull request by joshbaldwin**
I think this is about as good as it will get for this.  The only other idea would be to put all the symbolic link functionality into a function and then call that function either if the file already exists and we have the human readable parameter or at save_and_check().  But I wasn't sure if you want to put all the symbolic link creation at the very end.  If you are not opposed to this, I can make that adjustment.

Let me know if you have any other thoughts.

### CODE REVIEW

**Comment by kblin** [comment]
Hi Josh,
Thanks for the pull request, and sorry for the long delay in getting back to you. I was on vacation and tried to spend as little time at the PC as possible. :)

This looks pretty good already, but could you add a test for this functionality?

Also, https://landscape.io/github/kblin/ncbi-genome-download/pulls/27 tells me that some tabs snuck in there. Please replace them by spaces.

File: ncbi_genome_download/core.py [hunk 1]
```diff
@@ -142,11 +142,30 @@ def download_entry(entry, section, domain, output, file_format, human_readable):
         try:
             if has_file_changed(full_output_dir, parsed_checksums, fmt):
                 download_jobs.append(download_file(entry, full_output_dir, parsed_checksums, fmt, symlink_path))
+	    elif human_readable:
```
Comment (kblin) [inline ncbi_genome_download/core.py]: I think these two lines are the ones with tabs.

**Comment by kblin** [comment]
Great, thanks for the update. This just misses some tests for the new functionality and we're good to go.

**Comment by joshbaldwin** [comment]
These tests are turning out to be trickier than the code!  I'll have them complete soon.

**Comment by kblin** [comment]
Superseded by 41e4562123bbe7da2db9932675f4b276696dd4e4. Many thanks for this prototype, Josh. It was really valuable to figure out how I wanted to have this feature implemented.

**Pull request closed without merging**